app/styles/identity.css

```
@@ -1,58 +1,74 @@
+:root {
+    --button-verify-color: rgb(53, 194, 25);
+    --button-primary-color: rgb(81, 81, 245);
+    --button-copy-color: #e30062;
+}
+
 .identity-heading {
     margin-top: 25px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+    text-align: center;
 }
 
 .identity-container {
     display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
+    gap: 1rem;
     width: 100%;
 }
 
 .input-div {
-    width: 30%;
-}
-
-.input-url {
-    height: 2rem;
-    margin-top: 25px;
-    width: 60%;
-}
-
-.chaincode-div {
-    margin-top: 20px;
-    margin-left: 30%;
-    width: 50%;
+    display: flex;
+    gap: 1rem;
+    align-items: center;
 }
 
 .identity-Buttons {
-    display: flex;
-    background-color: rgb(81, 81, 245);
+    background-color: var(--button-primary-color);
     color: white;
     display: inline;
     cursor: pointer;
     border-radius: 3px;
     padding: 4px 15px;
 }
 
-.identity-label {
-    margin-top: 30px;
-    text-align: center;
+.copy-button {
+    margin-left: 0.75rem;
+    background-color: wheat;
+    color: var(--button-copy-color);
+    border: 2px solid var(--button-copy-color);
+    padding: 2px 5px;
 }
 
 .identity-verify-btn {
-    display: flex;
-    background-color: rgb(53, 194, 25);
+    background-color: var(--button-color);
     color: white;
     cursor: pointer;
     border-radius: 3px;
     padding: 8px 20px;
     border: none;
-    margin-top: 40px;
+    margin-top: 1rem;
+}
+
+.identity-label {
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    gap: 1rem;
+}
+
+.identity-check-label {
+    text-align: center;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    gap: 1rem;
+}
```

We can merge these CSS blocks. Also, the flex-direction value is  `row` by default. No need to explicitly provide it

```suggestion
.identity-label,
.identity-check-label {
    text-align: center;
    display: flex;
    align-items: center;
    gap: 1rem;
}
```